portfolio/src/main/java/com/google/sps/data/BookReader.java

```diff
@@ -0,0 +1,53 @@
+package com.google.sps.data;
+import com.google.sps.data.Book;
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Scanner;
+import javax.servlet.ServletException;
+
+/**
+ * Class that takes in a CSV FileName as the constructor parameter
+ * and populates a list of Books with title, genre, and reviews.
+ */
+public class BookReader {
```

Comment:
Can you please add a test case, including sample csv file(s). See: https://stackoverflow.com/questions/33666558/how-to-read-csv-file-from-resources-dir-in-junit-test-using-fileutils